.github/codeql/codeql-config.yml

```diff
@@ -2,4 +2,15 @@
 name: "CodeQL config"
 
 queries:
- - uses: security-extended  # security-and-quality (very verbose)
+ - uses: security-and-quality # (very verbose)
+
+paths-ignore:
```

Comment:
Apparently not supported for c/c++ code (yet?) but it doesn't hurt to leave it in.